ofrak_core/ofrak/core/elf/model.py

```diff
@@ -436,36 +405,17 @@ class ElfSectionHeader(ElfSectionStructure):
     def has_flag(self, flag: ElfSectionFlag) -> bool:
         return self.sh_flags & flag.value > 0
 
-    def get_flags(self) -> Iterable[ElfSectionFlag]:
+    def get_flags(self) -> Iterable[ElfSectionFlag]:  # TODO
         for flag in ElfSectionFlag:
             if self.has_flag(flag):
                 yield flag
 
-    def get_file_range(self) -> Range:
+    def get_file_range(self) -> Range:  # TODO
```

Comment:
Can delete by unwrapping any usages into this `Range.from_size` call

Reply:
Added a test so opted to keep this for now.